shopelectro/admin.py

```diff
@@ -287,6 +287,11 @@ def id_(self, obj):
         return obj.fake_order_number
 
 
+class MatrixBlockAdmin(mixins.PermissionsControl):
```

Comment:
matrix block should be inline like TagInline (shopelectro.admin.TagInline) for example

Reply:
found Artemiy's comment https://github.com/fidals/shopelectro/issues/934#issuecomment-508749154 about inline. With these restrictions separated model decision looks good